src/user/settings.rs

```diff
@@ -14,10 +14,14 @@ pub struct GlobalUserSettings {
     pub api_key: String,
 }
 
+fn default_project_type() -> ProjectType {
```

Comment:
this function should really be an implementation of the Default trait: https://doc.rust-lang.org/std/default/trait.Default.html

Reply:
Serde seem to support better default now too https://serde.rs/attr-default.html, but I wans't able to make them work. I will look into the Default trait.